P/PRIMA/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PRIMA"
+version = v"0.7.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/libprima/prima.git", "7565f6f42358062e56734a91fc62e0aa66ab7575")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd prima
+cmake -S . -B build -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
+cmake --build build --target install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+platforms = expand_gfortran_versions(platforms)
+platforms = filter(p -> libgfortran_version(p) != v"3", platforms)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libprimac", :libprimac),
+    LibraryProduct("libprimaf", :libprimaf)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
```

Comment:
As suggested by the build log:
```
┌ Warning: To ensure that the correct version of libgfortran is found at runtime, add the following entry to the list of dependencies of this builder
│
│     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
└ @ BinaryBuilder.Auditor /cache/julia-buildkite-plugin/depots/e2fd9734-29d8-45cd-b0eb-59f7104f3131/packages/BinaryBuilder/wIfP6/src/auditor/compiler_abi.jl:4
```

```suggestion
dependencies = Dependency[
    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
]
```